components/universal-widget/src/UniversalWidget.jsx

```diff
@@ -135,35 +160,73 @@ export default class UniversalWidget extends Widget {
         super.getWidgetChannelManager().unsubscribeWidget(this.props.id);
     }
 
+    /**
+     * Handle data received
+     * */
     handleWidgetData(data) {
-        if(data.data.length != 0){
+        if(data.data.length !== 0){
             this.setState({
                 metadata: data.metadata || this.state.metadata,
                 data: data.data
             })
         }
     }
 
-    render() {
-        return (
-            <div style={{ margin: 10, boxSizing: 'border-box' }}>
-                <VizG
+    /**
+     * get renderer type
+     * */
+    getRendererType() {
+        if (this.state.config
+            && this.state.config.charts
+            && this.state.config.charts.length > 0) {
```

Comment:
Entire if statement can go in one line. Doesn't exceed 120 rule.

Reply:
Fixed